pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -250,6 +250,16 @@ func (os *OpenStack) GetAttachmentDiskPath(instanceID, volumeID string) (string,
 	return "", fmt.Errorf("volume %s has no ServerId", volumeID)
 }
 
+//GetMaxVolLimit returns max vol limit
+func (os *OpenStack) GetMaxVolLimit() int64 {
+	if os.bsOpts.NodeVolumeAttachLimit > 0 && os.bsOpts.NodeVolumeAttachLimit <= 256 {
+		return os.bsOpts.NodeVolumeAttachLimit
+	}
+
+	return maxVol
```

Comment:
since you are changing this, can we rename this param? maxVol to maxNodeVolumeAttach or simiar?

Reply:
sure, will update it.